src/main/resources/templates/mdBenefitsFlow/applicantApplying.html

```diff
@@ -22,9 +22,9 @@
                               content=~{::inputContent})}">
                 <th:block th:ref="inputContent">
                   <th:block
-                      th:replace="~{fragments/inputs/radio :: radio(inputName='isApplicantApplying',value='true', label=#{applicant-applying.yes})}"/>
+                      th:replace="~{fragments/inputs/radio :: radio(inputName='isApplicantApplying',value='Yes', label=#{applicant-applying.yes})}"/>
```

Comment:
This fixes an issue with the mapping from this ticket: https://www.pivotaltracker.com/story/show/187040206